vendor/assets/javascripts/schedule/schedule.js

```diff
@@ -7,17 +7,28 @@ window.Schedule = function (scheduleContainer, options) {
   var BORDER_COLORS     = ['#ff2066', '#00aff2', '#ffcb45', '#48da58', '#d373da', '#a48363', '#ff9332'];
   var SELECTED_COLORS   = ['#ff3575', '#19b5f2', '#ffcf56', '#59dc68', '#d57fdd', '#ac8f71', '#ff9c46'];
   // /*                         PINK       BLUE       YELLOW     GREEN      PURPLE     BROWN      ORANGE */
+  var DAY_NAMES         = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
 
   // var NUM_COLORS        = 6;
   // var TEXT_COLORS       = ['#720', '#722', '#661', '#227', '#166', '#616'];
   // var BACKGROUND_COLORS = ['#fdc', '#edd', '#eed', '#dde', '#dee', '#ede'];
   // var BORDER_COLORS     = ['#b62', '#b44', '#994', '#448', '#499', '#949'];
 
   options = options || {};
+
+  // number of days to be shown
```

Comment:
Can we document these params using jsdoc instead of inline like we do for other public functions?